modules/ROOT/pages/amazon/amazon-dynamodb-connector-examples.adoc

```diff
@@ -0,0 +1,700 @@
+
+The XML for the Scan Table example should look like this:
+
+[source,text,linenums]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+
+<mule xmlns:ee="http://www.mulesoft.org/schema/mule/ee/core" xmlns:dynamodb="http://www.mulesoft.org/schema/mule/dynamodb"
+	xmlns:http="http://www.mulesoft.org/schema/mule/http"
+	xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:doc="http://www.mulesoft.org/schema/mule/documentation" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+http://www.mulesoft.org/schema/mule/dynamodb http://www.mulesoft.org/schema/mule/dynamodb/current/mule-dynamodb.xsd
+http://www.mulesoft.org/schema/mule/ee/core http://www.mulesoft.org/schema/mule/ee/core/current/mule-ee.xsd">
+	<http:listener-config name="HTTP_Listener_config" doc:name="HTTP Listener config" >
+		<http:listener-connection host="localhost" port="8081" />
+	</http:listener-config>
+	<dynamodb:config name="Amazon_DynamoDB_Configuration" doc:name="Amazon DynamoDB Configuration" >
+		<dynamodb:basic-connection accessKey="${config.accesskey}" secretKey="${config.secretkey}" />
+	</dynamodb:config>
+	<dynamodb:config name="Amazon_DynamoDB_Configuration1" doc:name="Amazon DynamoDB Configuration" >
+		<dynamodb:basic-connection accessKey="${config.accesskey}" secretKey="${config.seccretkey}" />
+	</dynamodb:config>
+	<flow name="delete-table-flow" >
+		<http:listener doc:name="Listener" path="/scantable" config-ref="HTTP_Listener_config"/>
+		<dynamodb:scan doc:name="Scan" config-ref="Amazon_DynamoDB_Configuration1" tableName="Issues"/>
+		<ee:transform doc:name="Transform Message" >
+			<ee:message >
+				<ee:set-payload ><![CDATA[%dw 2.0
+output application/json
+---
+{
+}]]></ee:set-payload>
+			</ee:message>
+		</ee:transform>
+		<logger level="INFO" doc:name="Logger" message="#[payload]"/>
+	</flow>
+</mule>
+----
+
+=== Response to the Scan Table Example
+
+The response to a `Scan` operation might look like this (with other sections of the response omitted for brevity):
+
+[source,xml,linenums]
+----
+{
+  "scannedCount": 2,
+  "lastEvaluatedKey": null,
+  "count": 2,
+  "consumedCapacity": null,
+  "items": [
+    {
+      "studentID": {
+        "ss": null,
+        "nullvalue": null,
+        "b": null,
+        "bool": null,
+        "ns": null,
+        "l": null,
+        "m": null,
+        "n": null,
+        "bs": null,
+        "s": "102"
+      },
+      ...
+  ]
+}
+----
+
+=== Run the Project
+
+Save and run the project as a Mule app:
+
+. In *Package Explorer*, right-click the project and click *Run As* > *Mule Application*.
+. Open a browser and check the response after you enter the `+http://localhost:8081/scantable+` URL.
++
+You should see the generated response from the `Scan` operation in the Mule console.
```

Comment:
```suggestion
You see the generated response from the `Scan` operation in the Mule console.
```

Reply:
GLOBALLY avoid modal verbs; see style ref.
GLOBALLY add "view" when preceding *Mule Palette* with "the."
GLOBALLY check with names and example names.